genesyscloud/task_management_workitem/resource_genesyscloud_task_management_workitem.go

```diff
@@ -0,0 +1,177 @@
+package task_management_workitem
+
+import (
+	"context"
+	"fmt"
+	"log"
+	resourceExporter "terraform-provider-genesyscloud/genesyscloud/resource_exporter"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/mypurecloud/platform-client-sdk-go/v119/platformclientv2"
+
+	"terraform-provider-genesyscloud/genesyscloud/consistency_checker"
+
+	gcloud "terraform-provider-genesyscloud/genesyscloud"
+
+	"terraform-provider-genesyscloud/genesyscloud/util/resourcedata"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/retry"
+)
+
+/*
+The resource_genesyscloud_task_management_workitem.go contains all of the methods that perform the core logic for a resource.
+*/
+
+// getAllAuthTaskManagementWorkitem retrieves all of the task management workitem via Terraform in the Genesys Cloud and is used for the exporter
+func getAllAuthTaskManagementWorkitems(ctx context.Context, clientConfig *platformclientv2.Configuration) (resourceExporter.ResourceIDMetaMap, diag.Diagnostics) {
+	proxy := newTaskManagementWorkitemProxy(clientConfig)
```

Comment:
Not super important, but I think this should be `getTaskManagementWorkitemProxy`

Reply:
We should fix this to be consistent.

Reply:
Fixed